client/src/router.tsx

```diff
@@ -0,0 +1,15 @@
+import { Children } from "react";
+import { createBrowserRouter } from "react-router-dom";
+import Authlayout from "./pages/layouts/Authlayout"
```

Comment:
lets have a space / new line btw 3rd party and relative imports for cleaner code